src/configuration.c

```diff
@@ -181,8 +189,6 @@ void free_configuration(void)
                 free(config.strValues[i]);
```

Comment:
The INI functions basically parse the INI file and store a `key`|`value` mapping with allocated keys and values, and the `k_free` `v_free` are both to `free`. It's sufficient to just `nc_hashmap_free` it. The methods return const pointers for hashmaps as the maps are assumed to own their contents (of course this is at the liberty of the `k_free` and `v_free` assignments)